lib/active_model/serializer/associations.rb

```diff
@@ -28,8 +27,8 @@ class << base
       end
 
       module ClassMethods
-        def inherited(base)
-          base._reflections = self._reflections.try(:dup) || []
+        def inherit_associations(base)
```

Comment:
Overriding the `inherited` method does not work when more than one modules do it.